design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -128,40 +130,63 @@ input GiftRegistryDynamicAttributeInput {
     value: String!
 }
 
-type CreateGiftRegistryOutput {
-    gift_registry: GiftRegistry
+interface GiftRegistryUserErrorsInterface {
```

Comment:
Same comment about removing "User" from the name.

Reply:
it's actually aligned with the rest of the schema:
https://github.com/magento/architecture/blob/master/design-documents/graph-ql/coverage/customer/Wishlist.graphqls#L84

I don't disagree that the "User" naming is redundant